mne/time_frequency/tfr.py

```diff
@@ -2092,15 +2092,47 @@ def __abs__(self):
         epochs.data = np.abs(self.data)
         return epochs
 
-    def average(self):
+    def average(self, method='mean'):
         """Average the data across epochs.
 
+        Parameters
+        ----------
+        method : str | callable
+            How to combine the data. If "mean"/"median", the mean/median
+            are returned.
+            Otherwise, must be a callable which, when passed an array of shape
+            (n_epochs, n_channels, n_time) returns an array of shape
+            (n_channels, n_time).
+            Note that due to file type limitations, the kind for all
+            these will be "average".
```

Comment:
brutal copy paste? aren't you supposed to have frequencies?

Reply:
Yep... :p